Dockerfile

```
@@ -5,12 +5,12 @@ COPY go.mod go.sum ./
 RUN go mod download
 
 COPY . .
-RUN go build -o parser cmd/parser/main.go
+RUN go build -o hermes cmd/hermes/main.go
 
 FROM alpine:latest
```

Hadolint (2.12.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)